src/modules/hip/kernel/blend.hpp

```diff
@@ -162,8 +162,12 @@ RppStatus hip_exec_blend_tensor(T *srcPtr1,
                                 T *dstPtr,
                                 RpptDescPtr dstDescPtr,
                                 RpptROIPtr roiTensorPtrSrc,
+                                RpptRoiType roiType,
                                 rpp::Handle& handle)
 {
+    if (roiType == RpptRoiType::LTRB)
+        hip_exec_roi_converison_ltrb_to_xywh(roiTensorPtrSrc, handle);
+
```

Comment:
make sure to use a #define for localThreads_x & .._y

Reply:
Done. Its now a common #define initialized in rpp_hip_common.